tools/specs/lib/optimizeGlbSpec.js

```diff
@@ -24,6 +24,38 @@ describe('optimizeGlb', function() {
             }), done).toResolve();
     });
 
+    it('optimizes and compresses a glb using the gltf-pipeline', function(done) {
```

Comment:
This description is a bit vague. Just `compresses textures in a glb using the gltf-pipeline` is fine.